arangodb/README-short.txt

```diff
@@ -0,0 +1 @@
+ArangoDB - a distributed open-source database with a flexible data model for documents, graphs, and key-values.
```

Comment:
This needs to be exactly 100 characters or less, how about this: `ArangoDB - a distributed open-source database with a flexible data model`